demos-europe/demosplan-core
# build(refs T19759): fix build

**Ticket:** https://yaits.demos-deutschland.de/T19759



### How to review/test

- run `yarn install`
- run `yarn dev:<project>`

-> there should be no errors


### PR Checklist
